components/PageButtons/src/index.css

```diff
@@ -0,0 +1,67 @@
+.denhaag-page-buttons {
+  display: flex;
+  flex-direction: column;
+  margin-block-start: var(--denhaag-page-buttons-margin-block-start);
+  margin-block-end: var(--denhaag-page-buttons-margin-block-end);
+}
+
+.denhaag-page-buttons__outline-dark-action {
```

Comment:
The focus is still bugged. Look at the implementation for Button. It uses an `::after` content style to create a focus border which goes around the button border, instead of overlapping it. I think we will want to apply the same style here.

Reply:
```
.denhaag-page-buttons__outline-dark-action.denhaag-button--focus:after,
.denhaag-page-buttons__outline-dark-action:focus:after {
  bottom: -3px;
  left: -3px;
  right: -3px;
  top: -3px;
}
```

I added this, that is the fix you where talking about right?

Reply:
Yep that's it. I can't see it in this PR though, did you push it?